Source/Core/Core/ActionReplay.cpp

```diff
@@ -327,6 +325,12 @@ static void LogInfo(const char* format, ...)
   }
 }
 
+template <typename... Args>
+static void LogInfo(std::string_view format, const Args&... args)
```

Comment:
Shouldn't this be `Args&&... args` and then perfectly forwarding `args...` to the call?

Reply:
This wouldn't result in any benefit as `make_format_args` uses as `const Args&...` as its parameter type